.travis.yml

```diff
@@ -1,17 +1,24 @@
 sudo: required
-
 language: java
-
 services:
-  - docker
-
+- docker
 script:
-  - mvn clean install -Ddocker.showLogs -Pdocker -Pjacoco -Pit -Pcoverage coveralls:report
-
+- mvn clean install -Ddocker.showLogs -Pdocker -Pjacoco -Pit -Pcoverage coveralls:report
 after_success:
-  - mvn deploy -DskipTests --settings .travis.settings.xml
+- mvn deploy -DskipTests --settings .travis.settings.xml
+deploy:
+ provider: script
+ script: 
+  - openssl aes-256-cbc -K $encrypted_acbbc88fb3ab_key -iv $encrypted_acbbc88fb3ab_iv -in gpg-sec.tar.enc -out gpg-sec.tar -d
+  - tar xvf gpg-sec.tar
+  - mvn deploy -DskipTests -Prelease -Ppassphrase --settings .travis.settings.xml
+ on:
+  tags: true
+  all_branches: true
```

Comment:
is it better to limit release to tag on master only?

Reply:
okay, I will update it